ansible/roles/ocp-workload-edge-deployments/files/project-iot-development.yaml

```diff
@@ -598,8 +599,10 @@ objects:
                           script {
                               openshift.withCluster() {
                                   openshift.withProject() {
-                                      // if everything else succeeded, tag the ${templateName}:latest image as ${testingProject}/${templateName}:latest
-                                      openshift.tag("${openshift.project()}/${templateName}:latest", "${testingProject}/${templateName}:latest")
+                                      // if everything else succeeded, tag the ${templateName}:latest image
+                                      // as ${testingProject}/${templateName}:latest
+                                      openshift.tag("${openshift.project()}/${templateName}:latest",
+                                      "${testingProject}/${templateName}:latest")
```

Comment:
This is how the jenkinsfile is written. I'm not certain you can just move this to another line. It is not a ansible or yaml multiline string.

This must be tested.

Reply:
Ah, you may be right. It didn't look like a yaml block, so I assumed that `//` was the comment char.

Reply:
Lines 602 and 603 are comments...but 604 & 605 are the code and I think that is what needs to be reverted since they are writing this jenkinsfile inline. Same with other comments.

Reply:
Doesn't groovy ignore line breaks? The code might look slightly different (have an extra line break), but would it not run the same?

Reply:
I am really not sure...hence my hesitation without either someone who knows for sure or with testing.